features/support/paths.rb

```diff
@@ -71,9 +68,9 @@ def path_to(page_name, *args) # rubocop:disable Metrics/AbcSize, Metrics/Cycloma
     when 'the billing information wizard page'
       edit_provider_admin_account_path(next_step: 'credit_card')
 
-    #
-    # Messages - Admin portal
-    #
+      #
+      # Messages - Admin portal
+      #
```

Comment:
Why? 😱

Reply:
This and the `find_by` "fixes" were auto-fixes by VScode :upside_down_face: 
I had Luis's permission to drop all of these, so I'll do it to avoid confusion :wink: